ussologin/visitwebpage.go

```diff
@@ -26,28 +26,31 @@ func VisitWebPage(filler form.Filler, client *http.Client, store TokenStore) fun
 	return func(u *url.URL) error {
```

Comment:
I'm wondering if there's a way to make different LoginMethods
methods compose decently - the step from "USSO-only" to "USSO-but-fallback-to-something-else"
is rather higher than I'd hope for.

One possibility might be something like:

     func VisitWebPageWithLoginMethods(filler form.Filler, client *http.Client, store TokenStore) func(*params.LoginMethods, *url.URL) error

Then a more generic function in idmclient could be tasked with composing
a set of (func(*params.LoginMethods, *url.URL) error) functions, getting
the LoginMethods value first, then calling each function in sequence until
one doesn't return ErrLoginMethodUnsupported.